lib/OrdinaryDiffEqFIRK/src/controllers.jl

```diff
@@ -24,8 +24,9 @@
     q
 end
 
-function step_accept_controller!(integrator, controller::PredictiveController, alg, q)
+function step_accept_controller!(integrator, controller::PredictiveController, alg::Union{RadauIIA3, RadauIIA5, RadauIIA9}, q)
```

Comment:
I think this restriction on the alg isn't desired.

Reply:
why? don't we have to specify which controller each algorithm should use?

Reply:
PredictiveController is not Radau-specific.

Reply:
the controller isn't set by dispatch. It's user configurable with the default set in `algorithms.jl` .